src/prefect/runner/runner.py

```diff
@@ -323,6 +332,8 @@ def goodbye_flow(name):
                 runner.start()
             ```
         """
+        signal.signal(signal.SIGTERM, self.handle_sigterm)
```

Comment:
do you think we should add a test for this?

Reply:
@zzstoatzz Yeah, I do, but we were having some troubles with that so we may need to add in a subsequent PR.